### DISCUSSION TITLE
Get the word boundaries for produced WAV

### DISCUSSION THREAD

**Discussion by begert** [coqui-ai/TTS#2133] [General Q&A]
Hi, I wonder if there is a way to get the word boundaries (times in the produced WAV) for some model with coqui. Searched the docs but the only output I found was a wav file. 
Would like to highlight one single word of the synthesized text while it is read. Is this possible? Any advice is appreciated.

**Comment by FrontierDK** [comment]
I would really appreciate this too. WAV files even support an EVNT chunk

More here: https://social.msdn.microsoft.com/Forums/security/en-US/bcd43193-8a6d-46a5-8fbb-fb972ea8dd5e/wav-files-with-evnt-chunks?forum=os_specifications
